remme/tests/__main__.py

```diff
@@ -17,7 +17,7 @@
 import unittest
 
 from remme.shared.logging import setup_logging
-from .test_token import *
+from .test_account import *
```

Comment:
Change to absolute imports. Tests are failed when a module is installed to the system (see [Circle CI logs](https://circleci.com/gh/Remmeauth/remme-core/180?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link))